### DISCUSSION TITLE
Change of index location in 0.74.X on Win10

### DISCUSSION THREAD

**Discussion by kirlat** [zim-desktop-wiki/zim-desktop-wiki#1991] [Q&A] [answered]
I've installed the 0.74.3 version of Zim and I really loved it. There are a lot of cool improvements! There is one thing, though, that creates an issue for me. As a result, I cannot use the new Zim notebooks on network drives. This is the placement of Zim's index files. 

I'm using Zim on Win10. My Zim notebooks are located on network drives mostly as they are shared across different users on different machines. The index files (`index.db`, `state.conf`) in 0.69.X versions are located in `C:\Users\[Windows User]\AppData\Roaming\zim\cache\zim\notebook-[Drive Letter]_[Zim Notebook Name]`. That works great. Each machine on a network has its own copy of an index file and it is accessible even if the network was slow/not available.

It seems that since 0.74.X the index files were moved to the `.zim` subfolder of the Zim notebook's folder. That creates an issue when the notebook is on the network drive. If the notebook was opened when the computer went to sleep and the machine wakes up, it may take a while for a network drive to become available. However, the opened Zim notebook tries to access the index files right after the computer is awaken, even if the netowrk is not ready yet and the network drive is not accessible. As a result, the index files cannot be found, Zim thinks that the notebook is empty and removes all pages from the page tree.

I'm wondering if the index files were relocated intentionally or was it a result of some Zim misconfiguration on my part. Is there any way to instruct Zim to place index files to local drives instead of network ones? If not, is there any way to avoid the problem with network drives described above?

Would really appreciate any advice. The new Zim 0.74.X is great, and I would really love to use it, but the abovementioned problem prevents that, unfortunately. It would be great if there be a way to solve it.

**Comment by jaap-karssenberg** [comment, marked as answer]
This behavior is determined by the value of the "shared" property in the
"notebook.zim" file - see https://zim-wiki.org/manual/Help/Config_Files.html
for more details.

-- jaap

**Reply by kirlat** [reply]
Thank you for your advice, Jaap! That solved the issue completely for me.
